### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by AlexPrusakov**
fixed typos

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,7 +25,7 @@
 </details>
 
 Discuss traditional approach to application development and its challenges and use cases.
-<details><summary>Traditiona Application development lifecycle</summary>
+<details><summary>Traditional Application development lifecycle</summary>
```
Comment (kirillkrylov) [inline README.md:28]: Great work !

File: README.md [hunk 2]
```diff
@@ -56,7 +56,7 @@ Discuss traditional approach to application development and its challenges and u
 <details><summary>Who are the citizens and who is citizen developer</summary>
 
 - Define citizen & citizen developer
-- How citizen developer differ from traditional developers
+- How citizen developers differ from traditional developers
```
Comment (kirillkrylov) [inline README.md:59]: thank you

Comment (AlexPrusakov) [reply README.md:59]: my pleasure

File: README.md (hunk 1 above)
Comment (AlexPrusakov) [reply README.md:28]: Eh, it was no big deal

**Pull request merged**